<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>readeleveWindow</class>
 <widget class="QMainWindow" name="readeleveWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>1021</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(17, 170, 119);
border-radius: 20px;
</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>901</width>
      <height>401</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>901</width>
        <height>291</height>
       </rect>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton{

border-radius:20px;


background-color: #ffbe62
}

#pushButton:hover{
border-radius:20px;
font: 10pt 'Century Gothic';
background:#ffffff;


}</string>
      </property>
      <property name="text">
       <string>Research</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>STUDENT_LIST</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>360</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton_2{

border-radius:20px;


background-color: #ffbe62
}

#pushButton_2:hover{
border-radius:20px;
font: 10pt 'Century Gothic';
background:#ffffff;


}</string>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>360</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton_6{

border-radius:20px;


background-color: #ffbe62
}

#pushButton_6:hover{
border-radius:20px;
font: 10pt 'Century Gothic';
background:#ffffff;


}</string>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>360</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton_7{

border-radius:20px;


background-color: #ffbe62
}

#pushButton_7:hover{
border-radius:20px;
font: 10pt 'Century Gothic';
background:#ffffff;


}</string>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="imprimer">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>360</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton_3{

border-radius:20px;


background-color: #ffbe62
}

#pushButton_3:hover{
border-radius:20px;
font: 10pt 'Century Gothic';
background:#ffffff;


}</string>
      </property>
      <property name="text">
       <string>imprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>901</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>360</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1020</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
